src/Rasa.Game/Structures/ClanMemberData.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Rasa.Structures
+{
+    public class ClanMemberData
```

Comment:
you can change this class to `public class ClanMemberData : IPythonDataStruct` and add read and write functions and then use is with`pw.WriteStruct(ClanMemberData ); in packets `SetClanMemberDataPacket ` `PlayerJoinedClanPacket ` and others